tests/test_hook_session_start_memory_prefetch.py

```
@@ -269,6 +726,81 @@ def _import_prefetch_module():
     return prefetch
 
 
+def _import_hook_repo_root_module():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_import_hook_repo_root_module`

(ANN202)